pandas/core/indexes/interval.py

```diff
@@ -201,6 +197,8 @@ class IntervalIndex(IntervalMixin, ExtensionIndex):
     _mask = None
 
     _data: IntervalArray
+    _values: IntervalArray
```

Comment:
umm, now i am confused, what is different about these?

Reply:
without this, mypy thinks _values is ExtensionArray and has a bunch of new complaints since we access `self._values.left` below

Reply:
kk, should try to remove this at some point